test/ttmlir/Dialect/TTNN/simple_broadcast.mlir

```diff
@@ -2,7 +2,8 @@
 #any_device_tile = #tt.operand_constraint<dram|l1|tile|none|interleaved|single_bank|height_sharded|width_sharded|block_sharded|any_layout|any_device_tile>
 module @jit_broadcast attributes {mhlo.num_partitions = 1 : i32, mhlo.num_replicas = 1 : i32} {
   func.func public @main(%arg0: tensor<1xf32> {mhlo.layout_mode = "default"}, %arg1: tensor<512x512xf32> {mhlo.layout_mode = "default"}) -> (tensor<512x512xf32> {jax.result_info = "", mhlo.layout_mode = "default"}) {
-    // CHECK: %[[C:.*]] = "ttnn.empty"[[C:.*]]
+    // CHECK: %[[C:.*]] = "ttnn.to_device"[[C:.*]]
+    // CHECK-NEXT: %[[C:.*]] = "ttnn.to_layout"[[C:.*]]
```

Comment:
I think these should be removed. We should clean unit tests in such way that we test IR only for specific things, in this case `ttnn.broadcast`. Reason is if someone comes and change something about `ttnn.to_layout` for example, they would have to change this unit test as well.